src/Microsoft.Azure.SignalR.Management/ServiceHubContextImpl.cs

```diff
@@ -55,7 +55,7 @@ public override async Task DisposeAsync()
             (ServiceProvider as IDisposable)?.Dispose();
         }
 
-        IInternalServiceHubContext IInternalServiceHubContext.WithEndpoints(IEnumerable<ServiceEndpoint> endpoints)
+        ServiceHubContext IInternalServiceHubContext.WithEndpoints(IEnumerable<ServiceEndpoint> endpoints)
```

Comment:
why change type?

Reply:
Otherwise we had to call `(serviceHubContext.WithEndpoints(..) as ServiceHubContext).NegotiateAsync()`